### DISCUSSION TITLE
Retrieve A specific Subtype of Asset Metadata only

### DISCUSSION THREAD

**Discussion by ujjyal** [Accenture/sfmc-devtools#1462] [Q&A] [answered]
I want to retrieve only the Template subtype of "Asset" metadata. Is it possible? What can I configure in mcdevrc.json file?
![Screenshot 2024-07-17 at 14 34 55](https://github.com/user-attachments/assets/57a5cf9e-5249-4db7-93d6-4d3565753339)

**Comment by JoernBerkefeld** [comment, marked as answer]
you can change it to say "asset-template" instead of "asset".
or you use `mcdev selectTypes` to do the same via menu.

**Reply by ujjyal** [reply]
Thank You. it worked.
